components/form-builder/app/Template.tsx

```diff
@@ -46,11 +46,13 @@ export const PageTemplate = ({
   title,
   navigation,
   leftNav = true,
+  autoWidth = false,
 }: {
   children: React.ReactNode;
   title: string;
   navigation?: React.ReactElement;
   leftNav?: boolean;
+  autoWidth?: boolean;
 }) => {
```

Comment:
`autoWidth` will always be defined because of the `autoWidth = false` in the function parameter list.

Reply:
You're right, it does need to be optional.  For some reason I thought I had used this pattern before and the linter had nagged me to change it...